WordPress/Classes/ViewRelated/System/Notices/NoticeView.swift

```diff
@@ -258,16 +258,17 @@ class NoticeView: UIView {
     }
 
     private enum Metrics {
-        static let cornerRadius: CGFloat = 13.0
+        /// The corner radius of the background (colored) and its shadow.
+        static let cornerRadius: CGFloat = 4.0
```

Comment:
I don't think we need this comment either

Reply:
It took me a while to scan the code to figure out how this is used. But yeah, I guess you're right. It's unnecessary noise since it's in the context of `NoticeView`.